Support vision model in chat app (#492)
microsoft/onnxruntime-genai: examples/chat_app/README.md

# set up

1. Install **onnxruntime-genai-directml** 

>     > If you want to use CUDA model, you can download `onnxruntime-genai-cuda` package.

Reviewer: ```suggestion
    > If you want to use a CUDA model, you can download the `onnxruntime-genai-cuda` package.
```

2. Download models to the created folder, take phi-3-mini directml as example.

>    > For CUDA model, only need to create a new folder and download model inside, e.g. `microsoft/Phi-3-vision-128k-instruct-onnx-cuda`.

Reviewer: ```suggestion
   > For a CUDA model, you only need to create a new folder and download the model inside (e.g. `microsoft/Phi-3-vision-128k-instruct-onnx-cuda`).
```

            --meta-llama_Llama-2-7b-chat-hf
            --mistralai_Mistral-7B-Instruct-v0.1
            ...
        --cuda

>             --phi3-mini-v

Reviewer: ```suggestion
            --phi-3-vision-128k-instruct
```

    ```

   Then open the local URL in broswer
   ![alt text](image.png)

>    For vision model, you will have blow UI interface.

Reviewer: ```suggestion
   For a vision model, you will have the below UI interface.
```
